qiskit/transpiler/preset_passmanagers/common.py

```diff
@@ -728,6 +728,7 @@ def get_vf2_limits(
     optimization_level: int,
     layout_method: Optional[str] = None,
     initial_layout: Optional[Layout] = None,
+    exact_match: bool = False,
```

Comment:
Do you expect this argument to remain useful even after `VF2PostLayout` is ported to Rust? Also, for consistency with other code, shouldn't we call this argument `strict`?

Reply:
This method is also called by things to use for `VF2Layout`, where `strict_direction=True` means quite different - that's what I was feeling awkward about and why I gave it a different name.  I don't feel super strongly.

It'll probably still have a meaning - the exact-match mode _is_ still a bit slower - but the limits will be far closer to the original ones.